src/test/java/org/symphonyoss/symphony/messageml/elements/form/PersonSelectorTest.java

```diff
@@ -65,10 +65,10 @@ public void sendValidPersonSelectorWithLabelAndTitleOnPresentationML() throws Ex
   }
 
   @Test
-  public void sendValidPersonSelectorWithSelectedUsersIds() throws Exception {
+  public void sendValidPersonSelectorWithValue() throws Exception {
     context.parseMessageML("<messageML>" +
             "  <form id=\"all-elements\">" +
-            "    <person-selector name=\"person-selector\" placeholder=\"Type a person's name\" selected-users-ids=\"123\"/>" +
+            "    <person-selector name=\"person-selector\" placeholder=\"Type a person's name\" value='[123]'/>" +
```

Comment:
You can put in your test value=\\"[123]\\" to be consistent with other parameters

or if you especially want to test with single quote, you can test also the other parameters :) `<person-selector name='person-selector' placeholder='Type a person's name' value='[123]'/>`

Reply:
changed it with double quotes! it was a typo :)